src/transformers/file_utils.py

```diff
@@ -263,6 +263,17 @@ def is_torch_cuda_available():
         return False
 
 
+_torch_fx_available = False
+if is_torch_available():
+    import torch
+
+    _torch_fx_available = version.parse(torch.__version__) >= version.parse("1.8")
```

Comment:
No direct import in this file to detect if a library is available please (that is how `import transformers` stays fast). I think this can be rewritten as:

```suggestion
if _torch_available:
    _torch_fx_available = version.parse(_torch_version) >= version.parse("1.8")
```